_posts/LeWagon/2023-07-15-Web-101.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
```

Comment:
It's unecessary to metion `layout post` since we have the following on _config.yml
```yml
defaults:
  -
    scope:
      path: ""
      type: "posts"
    values:
        layout: "post"
```